src/main.c

```diff
@@ -528,6 +528,20 @@ scrot_sel_and_grab_image(void)
   return im;
 }
 
+Imlib_Image
+scrot_grab_autoselect(void)
+{
+  Imlib_Image im = NULL;
+  int rx = opt.autoselect_x, ry = opt.autoselect_y, rw = opt.autoselect_w, rh = opt.autoselect_h;
+
+  scrot_do_delay();
```

Comment:
I also took the liberty to add the delay option (`-d, --delay NUM`) to the autoselect function.

Reply:
I am not sure if having the delay function auto applied is a good idea.

Reply:
The `scrot_do_delay()` function is called, but only actively affects the flow when the `-d` arg is passed.

Since it was a missing option with the autoselect flow, I added in case of someone wanting to use!

Reply:
Ah! Yeah, that makes sense.